torch/nested/_internal/ops.py

```diff
@@ -1112,6 +1125,26 @@ def _nested_get_ragged_idx(func, *args, **kwargs):
     return inp._ragged_idx
 
 
+@register_jagged_func(torch.ops.aten._nested_get_min_seqlen.default, "self: jt_all")
+def _nested_get_min_seqlen(func, *args, **kwargs):
+    _, new_kwargs = normalize_function(
+        func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
+    )
+
+    inp = new_kwargs.pop("input")
+    return inp._metadata_cache.get("min_seqlen", None)
+
+
+@register_jagged_func(torch.ops.aten._nested_get_max_seqlen.default, "self: jt_all")
+def _nested_get_max_seqlen(func, *args, **kwargs):
+    _, new_kwargs = normalize_function(
+        func, args=args, kwargs=kwargs, normalize_to_only_use_kwargs=True
+    )
+
+    inp = new_kwargs.pop("input")
+    return inp._metadata_cache.get("max_seqlen", None)
+
+
```

Comment:
Curious if it is possible to do this via an allow_in_graph'd function instead? 

Both would show up in the dynamic graph, and then aot autograd would end up desugaring them into accessing shape on inner tensor.

Reply:
If you're referring to the `_nested_get_{min,max}_seqlen()` ops, we also need access to them from C++ (for functional inverses / derivative formula), so I don't think an `allow_in_graph()`d function is enough.